modules/tasks/tasks/src/main/java/org/motechproject/tasks/service/impl/TaskServiceImpl.java

```diff
@@ -449,6 +456,7 @@ private TaskDataProvider findProviderByName(DataSource ds) {
     }
 
     private Set<TaskError> validateTrigger(Task task) {
+        LOGGER.debug("Validating the trigger in the task with name: {} and ID: {}", task.getName(), task.getId());
```

Comment:
This goes for all of these validations - we should log the results of validation.